tests/sample_data.py

```diff
@@ -70,4 +76,22 @@ def ask(self, answers, no_user_input=False):
         Choice(SOME_NAME, SOME_STRING, SOME_OPTIONS, SOME_DEFAULT),
         Confirm(SOME_NAME, SOME_STRING),
     ],
+    [
+        BasicQuestion(SOME_NAME, SOME_STRING, SOME_DEFAULT),
+        BasicQuestion(
+            SOME_NAME_WITH_SPACES, SOME_OTHER_STRING, SOME_NON_DEFAULT_OPTION
+        ),
+    ],
+]
+
+QUESTION_NAME_STANDARDIZATION_PARAMS = [
+    ["basic name", "foo", "--foo"],
+    ["name with one leading dash", "-foo", "--foo"],
+    ["name with trailing dash", "foo-", "--foo"],
```

Comment:
Perhaps add a case for leading/trailing underscores?

Reply:
Additional cases that are handled correcly, but could be added as test cases:
* double dashes are reduced to a single dash multiple times `foo--bar__baz`
* capital characters to lower case `Foo-Bar`